src/Collection.php

```diff
@@ -992,10 +992,6 @@ private function isOutInline($out)
 
         $out = (array) $out;
 
-        if (key($out) === 'inline') {
-            return true;
-        }
-
-        return false;
+        return key($out) === 'inline';
```

Comment:
This made me aware that we could stand to improve the logic in `isOutInline()`. I've pulled this change into my related PR (https://github.com/mongodb/mongo-php-library/pull/462) for posterity. Thanks!